### DISCUSSION TITLE
Vite should be included in the React section

### DISCUSSION THREAD

**Discussion by prince-of-maroc** [TheOdinProject/curriculum#25063] [Ideas]
Vite and Create React App are both incredibly simple. There is nothing to learn - in both cases its just a single command, either **npx create-react-app** or **npm create vite**. 

Why then is Vite not included in the React section? It is a build tool like CRA, only it saves far more space and runs exponentially faster. On mid-tier computers, CRA set-up can literally take several minutes. Vite on the other hand takes several seconds at most.

If CRA is taught for the purpose of "industry standard," then that makes sense, but I think there is no harm in at least mentioning Vite in passing.

**Comment by 01zulfi** [comment]
Vite is included in the [new React course](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new)
